### ISSUE TITLE
Box - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#5012]
Add Accessibility information to the Box page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/5004/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

When your work is ready for review please get a first pass review from another member of the accessibility sprint and once that is done reach out to Jessica to flag that a final review is needed.

Resources and videos for this work can be found here: https://hpe-my.sharepoint.com/shared?id=%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents%2FHPE%20Design%20System%2FAccessibility&listurl=https%3A%2F%2Fhpe%2Esharepoint%2Ecom%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents

**Comment by julauxen** [comment]
First go on the audit can be found here: https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=eHhVe1

**Comment by CillianHPE** [comment]
Added comments as a first pass to the Excel sheet.

In general, it isn't easy to know whether something is conditional or not applicable as the box is simply a container.

@jcfilben @julauxen

**Comment by britt6612** [comment]
@julauxen left some comments on the excel sheet.

**Comment by jcfilben** [comment]
@julauxen I went through as well and added some comments

**Comment by julauxen** [comment]
Thanks guys! @jcfilben I made some edits and left a few questions for you there.

**Comment by jcfilben** [comment]
@julauxen I did another pass on the excel sheet, I think I got everything answered but let me know if I missed any question. This is looking good though, you can go ahead and create a pull request for this.

**Comment by julauxen** [comment]
hey @jcfilben cool, I will create the PR but I still have some open questions. My biggest one is [this one.](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=hbwegn&nav=MTBfe2FiMjI4YzlmLTQyYzQtNDBkNC1iN2M1LTkyNTViMzczZmNmYn1fezhGRjRENDFGLTdGQzQtNEIzMC05NEIyLTg0NTI5OTQwRDQ3NH0)

**Comment by julauxen** [comment]
Here's the PR https://github.com/grommet/hpe-design-system/pull/5050

**Comment by jcfilben** [comment]
@julauxen added responses to your questions in the excel file

**Comment by jcfilben** [comment]
Great, I'll take a look

**Comment by britt6612** [comment]
closing since PR was merged and no grommet issues were required to be filed.
